BUILD.md

```diff
@@ -140,6 +144,24 @@ Getting the project to build and run on Windows is easy if you use Qt's IDE, Qt
     - If the project builds successfully it will run and the MultiMC5 window will pop up,
     - Test OpenSSL by making an instance and trying to log in. If Qt Creator couldn't find OpenSSL during the CMake stage, login will fail and you'll get an error.
 
+The following .dlls are needed for the app to run (copy them to build directory if you want to be able to move the build to another pc):
+```
+platforms/qwindows.dll
+libeay32.dll
+libgcc_s_dw2-1.dll
+libssp-0.dll
+libstdc++-6.dll
+libwinpthread-1.dll
+Qt5Core.dll
+Qt5Gui.dll
+Qt5Network.dll
+Qt5Svg.dll
+Qt5Widgets.dll
+Qt5Xml.dll
+ssleay32.dll
+zlib1.dll
+```
+
```

Comment:
Use `make install` instead. This is way too brittle.

Reply:
Honestly, I'm happy I managed to compile it once, and I don't want to spend more time on it - I just tried to clear up the readme a bit so that others don't have to spend as much time on it as I did.

Reply:
OK